abci/client/grpc_client.go

```diff
@@ -314,20 +314,59 @@ func (cli *grpcClient) ApplySnapshotChunkAsync(
 	)
 }
 
-func (cli *grpcClient) PrepareProposalAsync(
+// NOTE: call is synchronous, use ctx to break early if needed
```

Comment:
```suggestion
// ExtendVoteAsync ...
```

Also, if it's async then why is is synchronous?

Reply:
I honestly don't know why this comments says "call is synchronous", I just copied it from other methods.